docs/configuration/installation.rst

```diff
@@ -83,9 +47,14 @@ default. In order to install them, run:
 
     make install-postgres
 
+.. note::
+
+        The ``make`` commands are only available when Kinto was installed from
+        sources. The `underlying commands are available on Github
+        <https://github.com/Kinto/kinto/blob/master/Makefile#L22-L26>`_.
```

Comment:
Dont ever link to head directly, use a frozen commit hash from master history, so we're sure it won't be updated at the time users click that link.